docs/Antora/modules/getting_started/pages/quick_start_setting_up_csharp.adoc

```diff
@@ -0,0 +1,123 @@
+ * Alternatively (cross-) compiled projects can be run on https://learn.microsoft.com/en-us/windows/wsl/install[Windows Subsystem for Linux (WSL)] (use an Ubuntu 22.04 distribution).
+--
+====
+
+In general everything around .NET can be found here: +
+https://dotnet.microsoft.com/en-us/download/dotnet/8.0
+
+=== Creating and building the project
+
+In this guide we use a `Developer PowerShell for VS2022` prompt on Windows(TM) for creating and (cross-) building the project. +
+To run the result of the cross-compilation we use a `bash` prompt on Linux(TM) (or WSL on Windows(TM)).
+
+To start, create a `csharp/quick_start` directory and navigate to it.
+
+With  the following commands:
+
+- we first create an empty (.NET8.0) console application project
+- and then we add the _openDAQ.Net_ Bindings NuGet package reference (from nuget.org) to the project, which contains all the needed binaries.
+
+[source,shell]
+----
+dotnet new console
+dotnet add package openDAQ.Net
+----
+
+NOTE: Alternatively, the latest version of the .NET Bindings is available at https://docs-dev.opendaq.com/ and can be used via: +
+`dotnet add package openDAQ.Net --source path/to/folder` +
+See also https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-add-package[Learn NET - dotnet add package]
+
+With this, we are ready to start developing: +
+We fill in our `Program.cs` executable code as follows (replacing the default content):
+
+[source,csharp]
+----
+using Daq.Core.Types;
+using Daq.Core.Objects;
+using Daq.Core.OpenDAQ;
+
+// Create a fresh openDAQ(TM) instance, which acts as the entry point into our application
+Console.WriteLine("create instance...");
+var instance = OpenDAQFactory.Instance();
+
+Console.WriteLine();
+Console.WriteLine($"successfully created the instance '{instance.Name}'");
+
+Console.WriteLine();
+Console.Write("Press a key to exit the application ...");
+Console.ReadKey(intercept: true);
+----
+
+NOTE: When creating the openDAQ(TM) `instance`, we don't need to specify the root directory of our openDAQ(TM) Modules as they are conveniently being deployed by the NuGet package and found where they are.
+
+Now we can compile and run our program in our PowerShell(TM) instance from above.
+
+[tabs]
+====
+Windows::
++
+--
+[source,shell]
+----
+dotnet run
+----
+--
+
+Linux::
++
+--
+Cross-compiling on Windows(TM) and running the result on Linux(TM) using WSL:
+[source,shell]
+----
+dotnet build --runtime linux-x64
+wsl
+cd ./bin/Debug/net8.0/linux-x64/
+./quick_start
+exit
+----
+
+NOTE: `dotnet build --runtime linux-x64` builds the console application for Linux(TM) and uses the associated binaries from the NuGet package. The result (`./bin/Debug/net8.0/linux-x64/quick_start`) can then be run in WSL under Ubuntu 22.04 (see Requirements above).
+--
+====
+
+With the `run` command the project is being built (when there was a change after the last run) and the resulting `quick_start.exe` executed. +
```

Comment:
Delete the `+` at the end of line.

Reply:
no, I want a line-break here but not to start a new paragraph ;)